v1.1/build-a-python-app-with-cockroachdb-sqlalchemy.md

```diff
@@ -38,7 +38,11 @@ For other ways to install SQLAlchemy, see the [official documentation](http://do
 ## Step 5. Run the Python code
 
 The following code uses the [SQLAlchemy ORM](https://docs.sqlalchemy.org/en/latest/) to map Python-specific objects to SQL operations. Specifically, `Base.metadata.create_all(engine)` creates an `accounts` table based on the Account class, `session.add_all([Account(),...
-])` inserts rows into the table, and `session.query(Account)` selects from the table so that balances can be printed. Also note that the [cockroachdb python package](https://github.com/cockroachdb/cockroachdb-python) installed earlier is triggered by the `cockroachdb://` prefix in the engine URL.
+])` inserts rows into the table, and `session.query(Account)` selects from the table so that balances can be printed. 
+
+{{site.data.alerts.callout_info}}
+The [cockroachdb python package](https://github.com/cockroachdb/cockroachdb-python) installed earlier is triggered by the `cockroachdb://` prefix in the engine URL. Using `postgres://` to connect to your cluster will not work.
```

Comment:
Nice, @nstewart. But we can't use markdown inside these callouts. The link needs to be an `<a href>` and the backticks need to be replaced with `<code></code>`. Want me to take it?